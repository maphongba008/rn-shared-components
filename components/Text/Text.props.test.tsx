import React from "react";
import renderer from "react-test-renderer";
import { Text } from ".";

it("test parse box props", () => {
  const tree = renderer.create(
    <Text
      mMargin={8}
      mMarginTopLarge
      mMarginBottomLarge={false}
      mPaddingVerticalExtraLarge
      mTextAlignCenter
      mFont20
      mFontWeight200={false}
      mColor="blue"
      mBackgroundColor="green"
    >
      Hello world
    </Text>
  );

  expect(tree.toJSON()).toMatchSnapshot();
});
