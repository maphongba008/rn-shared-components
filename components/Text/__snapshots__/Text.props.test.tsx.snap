// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test parse box props 1`] = `
<Text
  style={
    Array [
      Object {
        "backgroundColor": "green",
        "color": "blue",
        "fontSize": 20,
        "margin": 8,
        "marginTop": 16,
        "paddingVertical": 32,
        "textAlign": "center",
      },
      undefined,
    ]
  }
>
  Hello world
</Text>
`;
